front/src/modules/study/components/AddOrEditStudyModal.tsx

```diff
@@ -256,15 +256,16 @@ export default function AddOrEditStudyModal({ editionMode, study }: Props) {
                       </div>
                     }
                     value={{
-                      id: currentStudy.study_type,
+                      id: currentStudy.study_type || '',
```

Comment:
You don't need this because an empty string is already falsy. 

Moreover, when selecting the "none" option, it currently gets 'nothingSelected' for the id (the e.target.value). To be consistent everywhere, we should have an id of empty string even for the selectoptions. You can start by replacing the StudyType like that : 


```
const study_types = [
  'nothingSelected',
  'timeTables',
  'flowRate',
  'parkSizing',
  'garageRequirement',
  'operationOrSizing',
  'operability',
  'strategicPlanning',
  'chartStability',
  'disturbanceTests',
] as const;

export type StudyType = typeof study_types;
```



```suggestion
                      id: currentStudy.study_type,
```